04_numeronym/jimbotech/numeronym_test.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+)
+
+type numeronymTest struct {
+	input    string
+	expected string
+}
+
+var numTests = []numeronymTest{
+	{"Kubernetes", "K8s"},
+	{"I love You ", "143"},
+	{"localizability ", "L12y"},
+	{"   Interoperability ", "i14y"},
+	{"", ""},
+	{"year 2038", "Y2K38"},
+	{"somebody said hello", "s17o"},
+}
+
+type multiTest struct {
+	input    []string
+	expected []string
+}
+
+var multiNumTests = []multiTest{
+	{[]string{"Kubernetes", "localizability", "Interoperability", "I love You ", "abc"},
+		[]string{"K8s", "L12y", "i14y", "143", "abc"}},
+}
+
+func TestWordMixer(t *testing.T) {
+
+	f := wordMixer()
+	for _, tt := range numTests {
+		actual := f(tt.input)
+		if actual != tt.expected {
+			t.Errorf("wordMixer(%v): expected %v, actual %v", tt.input, tt.expected, actual)
+		}
+	}
+}
+
+func TestNumeronyms(t *testing.T) {
+
+	for _, tt := range multiNumTests {
```

Comment:
Your table for this test has one entry. Using table-driven testing in that case makes no sense. Just use direct variables in the test function instead.

Reply:
Done

Reply:
Doesn't look done to me. But if you really want a 1-element table-driven test, go right ahead...